src/algebra/group/defs.lean

```diff
@@ -234,6 +234,12 @@ attribute [to_additive] group
 section group
 variables {G : Type u} [group G] {a b c : G}
 
+/-- `left_mul g` denotes left multiplication by `g` -/
+def left_mul : G → G → G := λ g : G, λ x : G, g * x
```

Comment:
Can you define it with the weaker assumption `[has_mul G]` instead of `[group G]`? (We might want to use this in fields, for instance). And can you also register the additive version, and `to_additive` from one to the other?

Reply:
I am not really sure where to place it then as has_mul is defined in core that looks not to be in mathlib

Reply:
The place where you have put it is perfect, just rename the section to `has_mul`, replace `[group G]` with `[has_mul G]`, and you're good to go!